browser/launcher.js

```diff
@@ -0,0 +1,23 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var _server;
+    var _browser;
+
+    function configure(options) {
```

Comment:
I think you should actually make this a constructor function, something like:

```js
function Launcher(options) {

}

Launcher.prototype.launch = function(url) {
  var server = this.server;
  var browser = this.browser;

  server.serveLiveDoc(url, function(err, url) {
    if (err) throw err;
    browser.update(url);
  });
};
...
exports.Launcher = Launcher;
```

And then when you want to use this, make a new instance of `Launcher`.